### PR TITLE
add package.json script for running JS tests in watch mode

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

none

#### What's this PR do?

what it says on the can

#### How should this be manually tested?

you should be able to do `npm run watch` in your docker container. the tests should run in watch mode, and should all be passing. everything else should be unchanged.

### CODE REVIEW

File: static/js/global_init.js [hunk 1]
```diff
@@ -79,7 +79,11 @@ process.on('unhandledRejection', reason => { // eslint-disable-line no-unused-va
 
 // fix 'unknown prop' error
 import injectTapEventPlugin from 'react-tap-event-plugin';
-injectTapEventPlugin();
+
+try {
```
Comment (noisecapella) [inline static/js/global_init.js]: Why was this needed?

Comment (aliceriot) [reply static/js/global_init.js]: it throws an error if it's already been called

Comment (aliceriot) [reply static/js/global_init.js]: (and this file is basically re-required every time mocha goes to run the tests)

Comment (noisecapella) [reply static/js/global_init.js]: Can you add a comment to explain the try block?

**Comment by aliceriot** [comment]
@noisecapella comments addressed
